packages/cozy-realtime/src/CozyRealtime.js

```diff
@@ -206,7 +206,7 @@ class CozyRealtime {
   _resubscribe() {
     this._retryLimit--
 
-    const subscribeList = Object.keys(this._events)
+    const subscribeList = this._getEventKeys(this._events)
       .map(key => {
         if (!key.includes(INDEX_KEY_SEPARATOR)) return
         if (this._events[key].length === 0) return
```

Comment:
A retirer surement aussi vu qu'on part de this._events

Reply:
Si je retire cette ligne les tests cassent.